src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -165,28 +168,40 @@ public static function createConnection(string $dsn, array $options = []): \Redi
 
         $params += $query + $options + self::$defaultConnectionOptions;
 
-        if (isset($params['redis_sentinel']) && !class_exists(\Predis\Client::class) && !class_exists(\RedisSentinel::class)) {
-            throw new CacheException(sprintf('Redis Sentinel support requires the "predis/predis" package or the "redis" extension v5.2 or higher: "%s".', $dsn));
+        if (isset($params['redis_sentinel']) && !class_exists(\Predis\Client::class) && !class_exists(\RedisSentinel::class) && !class_exists(Sentinel::class)) {
+            throw new CacheException(sprintf('Redis Sentinel support requires one of: ("predis/predis:*", "ext-redis:>=5.2", "ext-relay:*"): "%s".', $dsn));
```

Comment:
`Redis Sentinel support requires one of "predis/predis", "ext-redis >= 5.2" or "ext-relay": "%s".`